SAMtags.tex

```diff
@@ -234,13 +239,39 @@ \subsection{Barcodes}
 
 \begin{description}
 \item[BC:Z:\tagvalue{sequence}]
-Barcode sequence, with any quality scores stored in the {\tt QT} tag.
-
-\item[QT:Z:\tagvalue{qualities}]
-Phred quality of the barcode sequence in the {\tt BC} (or {\tt RT}) tag. Same encoding as {\sf QUAL}.
+Barcode sequence (Identifying the sample/library), with any quality scores (optionally) stored in the {\tt QT} tag.
+The {\tt BC} tag should match the {\tt QT} tag in length. 
+In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the barcodes and qualities and places a dash (`{\tt -}') between the barcodes from the same template. 
+
+\item[BX:Z:\tagvalue{sequence}]
+Sequence bases from the unique molecular identifier. 
+These could be either corrected or uncorrected. Unlike MI, this may be non-unique in the file. 
+Should comprise of sequence base possibly separated by hyphens as {\tt RX}.
+In case that the bases represent the corrected bases, the original sequence can be stored in {\tt RX} (similar to OQ storing the original qualities of bases.)
+
+\item[MI:Z:\tagvalue{str}]
+Molecular Identifier. 
+A unique ID within the SAM file for the source molecule from which this read is derived. 
+All reads with the same MI tag represent the group of reads derived from the same source molecule. 
+
+\item[QT:Z:\tagvalue{qualities}] 
+Phred quality of the sample-barcode sequence in the {\tt BC} (or {\tt RT}) tag. 
+Same encoding as {\sf QUAL}, i.e., Phred score + 33.
+See {\tt BC} for more detail.
+
+\item[QX:Z:\tagvalue{qualities}] 
+Phred quality of the unique molecular identifier sequence in the {\tt RX} or {\tt BX} tag. 
+Same encoding as {\sf QUAL}, i.e., Phred score + 33.
+The {\tt RX} and {\tt BX} tags should match the {\tt QX} tag in length. 
+In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the barcodes and qualities with no separator between the different barcodes.
```

Comment:
@dkj I know that this how 10X uses it, but IMHO the -1 -2 are redundant. since they inform the library which is known form the RG tag, we do not need it in the BX tag.